packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -396,12 +396,23 @@ class _IOSDeviceLogReader extends DeviceLogReader {
     });
   }
 
+  /// Blacklist of log output lines to suppress.
+  ///
+  /// Log lines begginning with strings in this list are suppressed from iOS
+  /// device log output.
+  static const List<String> _logBlacklist = const <String>[
```

Comment:
I'd originally done it as a local but changed my mind at the last minute, though really it was just to add docs, and I can do a simpler job of that as a plain comment inline. Done.